exporters/otlp/test/otlp_metrics_serialization_test.cc

```diff
@@ -43,6 +43,33 @@ static metrics_sdk::MetricData CreateSumAggregationData()
   return data;
 }
 
+static metrics_sdk::MetricData CreateUpDownCounterAggregationData()
+{
+  metrics_sdk::MetricData data;
+  data.start_ts = opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now());
+  metrics_sdk::InstrumentDescriptor inst_desc = {"UpDownCounter", "Robot Pose Y", "Meter",
+                                                 metrics_sdk::InstrumentType::kUpDownCounter,
+                                                 metrics_sdk::InstrumentValueType::kDouble};
+  metrics_sdk::SumPointData s_data_1, s_data_2;
+  s_data_2.value_ = 1.35;
```

Comment:
Can we change this to `s_data_1.value_ = 1.35;`

This seems to be copied from SumAggregation part, I can fix that later.